src/DataWrangling/metadata_field.jl

```diff
@@ -150,6 +150,8 @@ end
 struct ShiftSouth end
 struct AverageNorthSouth end
 
+reversed_sign(dataset, val_name) = false
+
```

Comment:
do we need the default? it might create issues?
was thinking that each submodule should define this

Reply:
This follows the design of what we do for temperature units and position in space. 
However, I was also thinking that a cleaner way to implement data mangling to comply to ClimaOcean conventions would be to have it done for submodules (i.e. remove also the general `temperature_units` and push it in the `EN4` module as well as the averaging for the u-velocity).

Reply:
what does "reversed sign" mean? It uses the same concept as "reversed vertical axis"? But I don't think it is refering to the function `reverse`, is it?

Reply:
right it was just the sign (from - to + and from + to minus). However, I removed it now that the convention for downwelling radiation is positive.

Reply:
Nevermind once again, ECCO's convention is aligned with ours, no need to switch signs... 😅